murenn/dtcwt/transform1d.py

```diff
@@ -95,9 +100,9 @@ class DTCWTDirect(DTCWT):
         alternate_gh (bool): If True (default), alternates between filter pairs
             (h0, h1) and (g0, g1) depending on odd vs. even wavelet scale j.
             Otherwise, uses (h0, h1) only. See Selesnick et al. 2005 for details.
-        padding_mode (str): One of 'zeros'(defalt), 'reflect', 'replicate', 
-            and 'circular'. Padding scheme for the filters. 
-        normalize (bool): If True (default), the output will be normalized by a 
+        padding_mode (str): One of 'zeros'(defalt), 'symmetric', 'replicate',
```

Comment:
typo : default (i'm sorry! )
if it would be better to use symmetric padding as default mode?